test-integration/filter-curve.js

```diff
@@ -0,0 +1,67 @@
+/* global artifacts */
+
+const ArgentContext = require("../utils/argent-context.js");
+
+const CurvePool = artifacts.require("CurvePoolMock");
```

Comment:
At some point it would be cool to be able to remove all these mock sources from the repo by working with precompiled artifacts. We currently do this only for a few Uniswap contracts (see `./scripts/provision_lib_artifacts.sh`) and for some of our legacy contracts (see `./test/basewallet.js`), but we could extent that to all third party contract dependencies.

Reply:
Good point, will add this to my todo